website/docs/components/stepper/nav/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Stepper Nav
+description: >-
+  A visual indicator of an on-page linear progress, where users can see where they are in a given flow.
+caption: >-
+  A visual indicator of an on-page linear progress, where users can see where they are in a given flow.
+links:
+  figma: >-
+    https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/branch/whv92tH8FWhxb4kk6Ip3qp/HDS-Components-v2.0?m=auto&node-id=72931-2356&t=A4lORtbini1AMxQN-1
+  github: >-
+    https://github.com/hashicorp/design-system/tree/main/packages/components/src/components/hds/stepper
+previewImage: assets/illustrations/components/stepper.jpg
+navigation:
+  keywords:
+    - page level
+    - progress
+    - progress bar
+    - steps
+    - tracker
```

Comment:
Should also include related components here

Reply:
+1, it doesn't look like these got added.